### PR TITLE
Support subset of Markdown in field descriptions

### PR DESCRIPTION
**Pull request by twschiller**
## What does this PR do?

- Adds support for subset of Markdown (links, bold, underline, strikeout, etc.) in field descriptions for brick and mod configuration
- Adds a MarkdownInline component
- Modifies FieldTemplate to use MarkdownInline
- Updates SchemaTree (used in brick reference in the Workshop and Add Brick Modal) to use a DescriptionCell which uses MarkdownInline
- Drops unused linkify-urls library we had previously used for linkifying URLs

## Reviewer Tips

- Read the components above first: MarkdownInline, FieldTemplate, SchemaTree
- Test snapshot changes are due to: 1) wrapping in span, 2) some tests snapshot the React component call with dangerouslySetInnerHTML vs. the produced DOM elements

## Discussion

- The FormBuilder supports full Markdown, see our implementation of [DescriptionField](https://github.com/pixiebrix/pixiebrix-extension/blob/03329626d8bbeaccc830cfff348376022786d87d/src/components/formBuilder/DescriptionField.tsx#L40-L40)

## Demo

- https://www.loom.com/share/00bcd1d8a52d422db87a7867558f3924

## Future Work

- Need to add support for markdown on the marketplace listing pages for bricks before we use markdown in any brick input/output schemas. GitHub issue here: https://github.com/pixiebrix/pixiebrix-www/issues/766
- Consider enabling more HTML tag types

## Checklist

- [x] Add tests
- [x] Designate a primary reviewer: @turbochef

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -88,7 +88,6 @@
     "json-stringify-safe": "^5.0.1",
     "jsonpath-plus": "^7.0.0",
     "kbar": "^0.1.0-beta.40",
-    "linkify-urls": "^4.0.0",
```
Comment (twschiller) [inline package.json:91]: Import is now unused

File: src/__mocks__/@/components/MarkdownInlineLazy.tsx [hunk 2]
```diff
@@ -15,25 +15,6 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import linkifyUrls from "linkify-urls";
```
Comment (twschiller) [inline src/__mocks__/@/components/MarkdownInlineLazy.tsx:18]: This is a bad git match. Dropping this file that was used for description generation previously

File: src/components/schemaTree/SchemaTree.tsx [hunk 3]
```diff
@@ -163,32 +201,6 @@ const SchemaTree: React.FunctionComponent<{ schema: Schema }> = ({
     return getFormattedData(schema);
   }, [schema]);
 
-  const columns = useMemo(
```
Comment (twschiller) [inline src/components/schemaTree/SchemaTree.tsx:166]: Extracted constant to module

File: src/__mocks__/@/components/MarkdownLazy.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import Markdown from "@/components/Markdown";
```
Comment (twschiller) [inline src/__mocks__/@/components/MarkdownLazy.tsx:18]: Mocking the "Lazy" versions of the imports so tests don't have to worry about the async import lifecycle

File: src/components/__snapshots__/MarkdownInline.test.tsx.snap [hunk 5]
```diff
@@ -0,0 +1,161 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`MarkdownInline linkifies text 1`] = `
```
Comment (BLoe) [inline src/components/__snapshots__/MarkdownInline.test.tsx.snap:3]: Bad snapshot here, make sure to call `.asFragment()`

File: src/pageEditor/pageEditor.html [hunk 6]
```diff
@@ -23,6 +23,8 @@
     <meta charset="utf8" />
     <link rel="stylesheet" href="loadingScreen.css" />
     <link rel="stylesheet" href="pageEditor.css" />
+    <!-- Open all links in new tab, never in the page editor #851 -->
+    <base target="_blank" />
```
Comment (BLoe) [inline src/pageEditor/pageEditor.html:27]: 👍

**Review by BLoe — COMMENTED** [review]
LGTM just need to fix the snapshots with `.asFragment()`

**Review by turbochef — APPROVED** [review]

**Pull request merged**